python/tests/test_nessie_client.py

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Did we not want to keep the shebang lines?

Why would we remove it from this file?